english/plugh/plugh.txt

```diff
@@ -0,0 +1 @@
+Suddenly, you find yourself on a hill looking at a small building.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.